igor-web/src/main/groovy/com/netflix/spinnaker/igor/build/model/GenericBuild.groovy

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2016 Schibsted ASA.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.igor.build.model
+
+class GenericBuild {
```

Comment:
@lwander I have been thinking about this a while, I want to make a separate pull request when this one is merged where I make jenkins use the same classes when shipping events to echo and sharing data in the info and build controller. I am worried about making too many big changes in this PR.